runtime-watcher/.goreleaser.yml

```diff
@@ -0,0 +1,60 @@
+# Documentation for how to configure goreleaser can be found at http://goreleaser.com
+project_name: kyma
+release:
+  github:
+    owner: kyma-project
+    name: runtime-watcher
+  prerelease: auto
+before:
+  hooks:
+    - make resolve
+builds:
+  - env:
+      - CGO_ENABLED=0
+    main: ./runtime-watcher/
+    goos:
+      - darwin
+      - linux
+      - windows
```

Comment:
I dont think we need to release the watcher image for all three OS. Linux should be enough, since that is where the watcher image will run later - WDYT?